web/src/main/webapp/WEB-INF/jsp/include/showTable.jsp

```diff
@@ -213,13 +213,13 @@
             <%-- Alter header format to reduce table space--%>
             <c:choose>
             	<c:when test="${column.name eq 'Date Updated'}">
-            		<c:out value="Date<br />Updated" escapeXml="false" />
+					<fmt:message key="date_br_updated" bundle="${resword}"/>
             	</c:when>
             	<c:when test="${column.name eq 'Last Updated by'}">
-            		<c:out value="Last<br /> Updated by" escapeXml="false" />
+					<fmt:message key="last_br_updated_by" bundle="${resword}"/>
             	</c:when>
             	<c:when test="${column.name eq 'Date Created'}">
-            		<c:out value="Date<br />Created" escapeXml="false" />
+					<fmt:message key="date_br_created" bundle="${resword}"/>
             	</c:when>
```

Comment:
These 3 changes don't look like they'll work given that the tests are against the hard coded English while the column names are often, if not always, already internationalized.  Will probably need to update the 'c:when test' to test against the property file's entry's value.